vividus-plugin-web-app-to-rest-api/src/main/java/org/vividus/crawler/LinkCrawler.java

```diff
@@ -33,14 +36,15 @@ public class LinkCrawler extends WebCrawler
 {
     private static final Logger LINK_CRAWLER_LOGGER = LoggerFactory.getLogger(LinkCrawler.class);
 
-    private static final String[] EXCLUDED_EXTENSIONS = { ".css", ".gif", ".gz", ".ico", ".jpeg", ".jpg", ".js",
-            ".mp3", ".mp4", ".pdf", ".png", ".zip" };
-
     private final LinkCrawlerData linkCrawlerData;
 
-    public LinkCrawler(LinkCrawlerData linkCrawlerData)
+    private final Set<String> allowedFileExtensions;
```

Comment:
my apologies, I've been misled, the final solution was to move **dis**allowed file extensions to properties